index.ios.js

```diff
@@ -1,9 +1,12 @@
-import Containers from './src/lib/containers'
-import Components from './src/lib/components'
-import Routes from './src/lib/relay/routes'
+// First things first.
+import "./src/lib/error_reporting"
```

Comment:
I can't find this file?

![screen shot 2017-05-12 at 09 02 23](https://cloud.githubusercontent.com/assets/49038/25988886/bbd89c54-36f1-11e7-918f-a086f9421481.png)